baggage/HTTP_HEADER_FORMAT.md

```diff
@@ -45,7 +42,8 @@ Producers SHOULD try to produce a `baggage-string` without any `list-member`s wh
 
 #### key
 
-A `token` which identifies a `value` in the `baggage`. `token` is defined in [RFC7230, Section 3.2.6](https://tools.ietf.org/html/rfc7230#section-3.2.6).
+A `key` which identifies a `value` in the `baggage`.
```

Comment:
This sentence feels a bit ambiguous (key identifying a value) and is also not complete; instead, can we remove this sentence and instead add to the definitions list something like:

key                    =  token ; as defined in `Name` in [[XML]]

and then in the ####key section maybe we can have:

`token` is defined as `Name` in [[XML]], Section 2.3: https://www.w3.org/TR/xml/#NT-Name.